Makefile

```diff
@@ -15,11 +15,11 @@
 PROJECT_ID ?= $(shell gcloud config get-value project)
 SHORT_SHA := $(shell git rev-parse --short=7 HEAD)
 BUILDER_IMG ?= gcr.io/${PROJECT_ID}/builder:${SHORT_SHA}
-CONTROLLER_IMG ?= gcr.io/${PROJECT_ID}/controller:${SHORT_SHA}
-RECORDER_IMG ?= gcr.io/${PROJECT_ID}/recorder:${SHORT_SHA}
-WEBHOOK_IMG ?= gcr.io/${PROJECT_ID}/webhook:${SHORT_SHA}
-DELETION_DEFENDER_IMG ?= gcr.io/${PROJECT_ID}/deletiondefender:${SHORT_SHA}
-UNMANAGED_DETECTOR_IMG ?= gcr.io/${PROJECT_ID}/unmanageddetector:${SHORT_SHA}
+CONTROLLER_IMG ?= gcr.io/${PROJECT_ID}/cnrm/controller:${SHORT_SHA}
+RECORDER_IMG ?= gcr.io/${PROJECT_ID}/cnrm/recorder:${SHORT_SHA}
+WEBHOOK_IMG ?= gcr.io/${PROJECT_ID}/cnrm/webhook:${SHORT_SHA}
+DELETION_DEFENDER_IMG ?= gcr.io/${PROJECT_ID}/cnrm/deletiondefender:${SHORT_SHA}
+UNMANAGED_DETECTOR_IMG ?= gcr.io/${PROJECT_ID}/cnrm/unmanageddetector:${SHORT_SHA}
```

Comment:
This change may be risky, you may want to double check the internal mirrored repo to see whether this path is used in the KCC release.

Reply:
As long a PROJECT_ID != config-management-release I think this should be safe. Would you be satisfied if we just exited with an error if PROJECT_ID == config-management-release?

Reply:
I think my question is: do we have any other processes (like prow, louhi or gob) that relies on these images. I don't know the answer. But every time a build change causes a release headache and a couple of hours additional work. 
We can go with checking the PROJECT_ID == config-management-release  and see if it still works

Reply:
I searched around, this affects where the KCC images are pushed, which is managed by kustomize https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/installbundle/components/deletiondefender/deletiondefender_image_patch_template.yaml#L23

Looks like in the internal repo, we push the image to  gcr.io/$PROJECT/cnrm/ of another GCP project. https://source.corp.google.com/h/cnrm/cnrm/+/main:google-internal/Makefile;l=28

Reply:
The key is that we expect images to be in $PROJECT/cnrm/. That is the address our charts use. So for local development to work, local deployments also need to write to those locations.